documentation/src/main/asciidoc/advanced-features.asciidoc

```diff
@@ -19,6 +20,22 @@ SearchFactory searchFactory = fullTextSession.getSearchFactory();
 ----
 ====
 
+[[AccessingSearchIntegrator]]
+=== Accessing the SearchIntegrator
+The SearchIntegrator is an object that gives access to lower level API of Hibernate Search.
```

Comment:
The interface `SearchIntegrator` gives access to lower level APIs of Hibernate Search.